Utilities/Config.h

```diff
@@ -521,4 +522,40 @@ namespace cfg
 
 		void from_default() override;
 	};
+
+	struct device_info
+	{
+		std::string path;
+		std::string serial;
+		std::string vid;
+		std::string pid;
+	};
+
+	class device_entry final : public _base
+	{
+		map_of_type<device_info> m_map{};
+		map_of_type<device_info> m_def{};
```

Comment:
I do think that explicitness is relevant. 
Could you please give it a name `m_default`?

Reply:
I don't see a reason to waste more time on something as obvious